lib/provider/volume_file_access_point.go

```diff
@@ -78,8 +78,8 @@ type VolumeAccessPointRequest struct {
 	//ResourceGroup for ENI
 	ResourceGroup *ResourceGroup `json:"resource_group,omitempty"`
 
-	//EncryptionInTransit
-	EncryptionInTransit string `json:"transit_encryption,omitempty"`
+	//TransitEncryption
+	TransitEncryption string `json:"transit_encryption,omitempty"`
```

Comment:
please make sure to use omitempty and required parameters wherever required, it should not be generic